migration/template/apply/90-apply.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
Should this file be renamed to `80-apply.sh` so all scripts that need to be after apply can be `9x-foobar`? 91-99 should be enough but it feels like it's a nice separation between before apply and after apply